cmd/templating-device.go

```diff
@@ -94,6 +94,9 @@ func (td templatingDevice) preRenderedTemplate(inputFile string) *template.Templ
 }
 
 func (td templatingDevice) spit(outputDir string) error {
+	if err := os.MkdirAll(outputDir, 0700); err != nil {
```

Comment:
why not 0600? Do we need the dir to have execute perms? I didn't think it's needed. Wondering why? Thanks.